aws/ci_build

```diff
@@ -160,6 +160,8 @@ def main
 
         # Schedule a reminder for the dotd to check Zendesk in 2 hours
         Slack.remind(Slack.user_id(DevelopersTopic.dotd), Time.now.to_i + 7200, 'Reminder: check <https://codeorg.zendesk.com/agent/filters/44863373|Zendesk>')
+        # Remind dotd to update the flaky test tracker on DTP as a once-ish a day reminder.
+        Slack.remind(Slack.user_id(DevelopersTopic.dotd), Time.now.to_i, 'Please update the <https://docs.google.com/spreadsheets/d/1DD6ebHM-2fsiMxxYCzhJYd1DzA69Z20qrCItaER9vto|Flaky Test Tracker>')
```

Comment:
Is this a one time reminder, or a recurring reminder?

Reply:
That was the best mechanism I could find. Ideally it would be once a day around 9:00 am PT, but I couldn't find a way to do that

Reply:
to do this once every morning, you could add this to bin/cron/update_dotd

Reply:
Perfect! Updated to put into update_dotd instead.

Reply:
Oh, I guess the ZenDesk reminder works the same way. I was worried this might create a daily reminder, such that on the third day, the current and two prior DOTD's would get reminders.